cfme/tests/cloud_infra_common/test_power_control_rest.py

```diff
@@ -40,15 +41,18 @@ def _delete_vm():
 
 def verify_vm_power_state(vm_obj, state, action=None):
     vm = vm_obj.get_vm_via_rest()
-    if vm_obj.provider.one_of(GCEProvider, EC2Provider):
-        num_sec = 1600  # extra time for slow providers
+    if vm_obj.provider.one_of(GCEProvider, EC2Provider, SCVMMProvider):
+        num_sec = 4000  # extra time for slow providers
     else:
         num_sec = 1000
     if action:
         action()
-    wait_for(lambda: vm.power_state == state,
-        num_sec=num_sec, delay=20, fail_func=vm.reload,
-        message='Wait for VM state `{}` (current state: {})'.format(state, vm.power_state))
+
+    def _state_changed():
```

Comment:
heh, you're right, it does.

Can I suggest removing this and replacing it with common.VM.wait_for_vm_state_change()?